README.md

```diff
@@ -48,6 +51,34 @@ pacman -S base-devel git ncurses lib32-zlib gawk time unzip perl-xml-libxml \
  flex wget gettext quilt python2 openssl
 ```
 
+
+### Build-System-Structure  
+
+Where can I change something? What does these files do? 
+
+```
+:: Settings
+
+config.mk             - select what package lists to build
+modules               - define git repository, revision, etc of OpenWRT and Packages
+
+patches/              - Patches against the OpenWRT directory 
+packagelists/         - Package Lists 
+profiles/             - List of router profiles for each architecture - names come from the Imagebuilder
+
```

Comment:
Why not mentioning "configs/" here?

Reply:
What does the files in `configs/` exactly do? It looks like they handle some kernel and package-foo...

Reply:
common.config: contains the OpenWrt config-options for all targets
target.config: gets added to the common.config and contains the target-specific config-options. possibly overwrites earlier settings.
common-autobuild.config: gets added on top , if IS_BUILDBOT=yes is defined

see: https://github.com/freifunk-berlin/firmware/blob/master/Makefile#L122 and following lines

Reply:
added in current version.